generator/templates/new/mix.exs

```diff
@@ -0,0 +1,37 @@
+defmodule <%= mod_name %>.Mixfile do
+  use Mix.Project
+
+  def project do
+    [app: :<%= app_name %>,
+     version: "0.0.1",
+     elixir: "~> 1.0",
+     elixirc_paths: elixirc_paths(Mix.env),
+     build_embedded: Mix.env == :prod,
+     start_permanent: Mix.env == :prod,
+     deps: deps]
+  end
+
+  # Configuration for the OTP application
+  #
+  # Type `mix help compile.app` for more information
+  def application do
+    [
+      applications: [:logger, :butler]
+    ]
+  end
+
+  def elixirc_paths(_), do: ["plugins"]
+
+  # Dependencies can be Hex packages:
+  #
+  #   {:mydep, "~> 0.3.0"}
+  #
+  # Or git/path repositories:
+  #
+  #   {:mydep, git: "https://github.com/elixir-lang/mydep.git", tag: "0.1.0"}
+  #
+  # Type `mix help deps` for more examples and options
+  defp deps do
+    [{:butler, "~> 0.3.0"}]
```

Comment:
I'd make that version string an interpolated variable based on the current version of butler (which should be available).